static/src/javascripts/projects/journalism/modules/submit-form.js

```diff
@@ -2,83 +2,128 @@
 import fastdom from 'lib/fastdom-promise';
 import fetch from 'lib/fetch';
 
-const isCheckbox = element => element.type === 'checkbox';
+const targetUrl = 'https://callouts.code.dev-guardianapis.com';
```

Comment:
Can `targetUrl` live in the configuration files? Or is it a temporary work-around for testing?

Reply:
good point. new change should fix this